src/OpenLoco/src/Vehicles/Vehicle.h

```diff
@@ -26,6 +26,16 @@ namespace OpenLoco::Vehicles
     constexpr auto kMaxRoadVehicleLength = 176;    // TODO: Units?
     constexpr uint8_t kWheelSlippingDuration = 64; // In ticks
 
+    enum class EngineStatus : uint8_t
```

Comment:
I think EngineStatus is a bit odd of a name, this seems more like a VehicleState

Reply:
I wanted to name it something less encompassing than all of vehicle, since it mostly is used to interface with the sound engine.

Reply:
I'm fine with either name

Reply:
I guess MotorState is fine